package.json

```diff
@@ -14,6 +14,7 @@
         "@wdio/selenium-standalone-service": "^6.11.0",
         "@wdio/spec-reporter": "^6.6.0",
         "autoprefixer": "^10.0.1",
+        "bootstrap-sass": "^3.4.1",
```

Comment:
Added in the sass version of bootstrap 3.4. We should perhaps discuss just generating our own customised version of bootstrap CSS with the KBase colour scheme so we don't have to have the bootstrap overrides and don't have to generate a load of new buttons and other UI elements.